docs/how-to-guides/openstack-swift.md

```diff
@@ -0,0 +1,26 @@
+Sometimes it is desirable to cache swift storage objects inside the charm for faster processing and reducing the number of requests to the swift server.
+
+This guide will demonstrate how to deploy this charm along with Openstack swift storage.
+
+First, connect to your openstack instance with your credentials, sourcing an .rc file:
```

Comment:
A reference about how to configure the credentials here might help.

Reply:
to do that we should make a tutorial on how to install the devstack instance probably, is that advisable? It would be a simple tutorial detailing 2 steps: execute some script to install everything locally and run some command to extract your credentials.

Reply:
I was thinking just a link to official OpenStack doc about that but this could be in the tutorial as well.